pypeit/spectrographs/p200_ngps.py

```diff
@@ -0,0 +1,619 @@
+            dataext         = 1, # All R channel images assigned to extension 1
+            specaxis        = 1,
+            specflip        = False, 
+            spatflip        = False, 
+            platescale      = 0.5, 
+            darkcurr        = 0.0,  # e-/pixel/hour (No dark current)
+            saturation      = 65000., # ???
+            nonlinear       = 40./45.,
+            mincounts       = -1e10, # check
+            numamplifiers   = 1, 
+            gain            = np.atleast_1d(2.8),
+            ronoise         = np.atleast_1d(8.5),
+            datasec         = np.atleast_1d(flip_fits_slice(hdu[1].header['DATASEC'])),
+            oscansec        = np.atleast_1d(flip_fits_slice(hdu[1].header['BIASSEC']))
+        )
+
+        # Instantiate
+        detector1 = detector_container.DetectorContainer(**detector_dict1)
+        return detector1
+
+
+    @classmethod
+    def default_pypeit_par(cls):
+        """
+        Return the default parameters to use for this instrument.
+        
+        Returns:
+            :class:`~pypeit.par.pypeitpar.PypeItPar`: Parameters required by
+            all of PypeIt methods.
+        """
+        par = super().default_pypeit_par()
+
+        par['calibrations']['slitedges']['sync_predict'] = 'nearest'
+        par['calibrations']['slitedges']['edge_thresh'] = 10. # Lower edge tracing thresdhold to catch leftmost slit
+
+        par['calibrations']['slitedges']['minimum_slit_length'] = 100 # Set minimum slit length 
+
+        par['calibrations']['slitedges']['add_slits'] = ['1:2090:24:153']
+        
+        par['scienceframe']['process']['combine'] = 'median'
+        par['calibrations']['standardframe']['process']['combine'] = 'median'
+
+        par['scienceframe']['process']['use_overscan'] = True
+        par['scienceframe']['process']['sigclip'] = 4.0
+        par['scienceframe']['process']['objlim'] = 5.0
+
+        # Make a bad pixel mask
+        par['calibrations']['bpm_usebias'] = True
+
+        # Set pixel flat combination method
+        par['calibrations']['pixelflatframe']['process']['combine'] = 'median'
+
+        par['calibrations']['wavelengths']['lamps'] = ['ThAr'] # FeAr and ThAR lamps for NGPS (ThAr)
+        par['calibrations']['wavelengths']['method'] = 'full_template' # Use wavelength template
+        par['calibrations']['wavelengths']['reid_arxiv'] = 'wvarxiv_p200_ngps_20250131T1227.fits' # R Channel
+
+        par['calibrations']['wavelengths']['rms_thresh_frac_fwhm'] = 1.0 
+
+        # Do not flux calibrate
+        #par['fluxcalib'] = None
+        par['sensfunc']['algorithm'] = 'UVIS' # CHANGE #
+
+        # Set the default exposure time ranges for the frame typing
+        par['calibrations']['biasframe']['exprng'] = [None, 0.001]
+        par['calibrations']['darkframe']['exprng'] = [999999, None]     # No dark frames
+        par['calibrations']['pinholeframe']['exprng'] = [999999, None]  # No pinhole frames
```

Comment:
I don't think these two lines are necessary, given that `check_frame_type` skips these two frame types anyway.